### PR TITLE
linter-config for dotenv-linter

### PR DESCRIPTION
**Pull request by bitbirddev**
for more information visit  [dotenv-linter](https://github.com/dotenv-linter/dotenv-linter)

### CODE REVIEW

File: lua/lint/linters/dotenv_linter.lua [hunk 1]
```diff
@@ -0,0 +1,22 @@
+return {
+  cmd = function()
+    local local_executable = vim.fn.fnamemodify("./bin/dotenv-linter", ":p")
+    local stat = vim.loop.fs_stat(local_executable)
+    if stat then
+      return local_executable
+    end
+    return "dotenv-linter"
+  end,
```
Comment (mfussenegger) [inline lua/lint/linters/dotenv_linter.lua:9]: This looks rather custom, or is this a established convention for `dotenv-linter` ?

File: README.md [hunk 2]
```diff
@@ -23,7 +23,6 @@ available language server do.
   - If using [vim-plug][3]: `Plug 'mfussenegger/nvim-lint'`
   - If using [packer.nvim][4]: `use 'mfussenegger/nvim-lint'`
 
-
```
Comment (mfussenegger) [inline README.md:26]: Please no unrelated style changes

Comment (bitbirddev) [reply README.md]: i am sorry. i've just revertes the changes!

File: lua/lint/linters/dotenv_linter.lua (hunk 1 above)
Comment (bitbirddev) [reply lua/lint/linters/dotenv_linter.lua]: ok, done :)
